generators/java/okhttp-gson/templates/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -1602,14 +1602,18 @@ public class ApiClient {
     public RequestBody buildRequestBodyMultipart(Map<String, Object> formParams) {
         MultipartBody.Builder mpBuilder = new MultipartBody.Builder().setType(MultipartBody.FORM);
         for (Entry<String, Object> param : formParams.entrySet()) {
-            if (param.getValue() instanceof File) {
```

Comment:
IMO not supporting File objects is a regression. This code path should still support it, i.e. by generating a FileTransfer object from it.

Reply:
We can keep this code, but it's going to be dead code since all the `File` objects in API methods has been replaced with `FileTransfer` ones, so it will never happen to have a `File` at this point..

This is not a regression, user can still easily provide a `File` object wrapping that in a `FileTransfer` one. And, anyway, [old library](https://github.com/onfido/onfido-java/blob/e92cb1017a74fff58877f8a2ad0be06a7f4725d1/onfido-java/src/main/java/com/onfido/LivePhotoManager.java#L39) was not allowing providing File object at all, but just an `InputStream` object.